rails/Gemfile

```diff
@@ -0,0 +1,10 @@
+source 'https://rubygems.org'
+
+gem 'benchmark-ips', github: 'kirs/benchmark-ips', branch: 'quiet-mode'
```

Comment:
What is different between your `benchmark-ips` and evans? it looks like you're missing some commits he has pushed.